lib/spec/openapi/utils.js

```diff
@@ -35,14 +35,15 @@ function prepareDefaultOptions (opts) {
     transform,
     hiddenTag,
     extensions,
-    hideUntagged
+    hideUntagged,
+    ...openapi
```

Comment:
Why spread operator at below?

Reply:
Good point, I'm now overriding all the default `null` values for `info`, `servers`, etc. Is it okay if I use spread at the top? It diverges from the pattern that was set before me. Is there a reason `info` etc. have to be set to `null`?